src/components/shared/eventlist/eventlist.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useStaticQuery, graphql } from 'gatsby';
+
+import Box from 'components/shared/box';
+import FeaturedEventCard from 'components/shared/featuredeventcard';
+import EventCard from 'components/shared/eventcard';
+
+const EventList = () => {
+  const data = useStaticQuery(graphql`
+    query UpcomingEvents {
+      allMeetupEvent {
+        edges {
+          node {
+            id
+            name
+            simple_html_description
+            local_date
+            short_link
+            time
+            featured_photo {
+              photo_link
+            }
+            venue {
+              name
+              address_1
+              city
+              state
+              zip
+            }
+          }
+        }
+      }
+    }
+  `);
+
+  const [nextEvent, secondEvent, ...otherEvents] = data.allMeetupEvent.edges;
+  const events = otherEvents.map((edge, index) => (
+    <EventCard key={index} event={edge.node} />
```

Comment:
MICRO OPTIMIZATION... but `key={edge.id}` is generally better practice!

Reply:
Yep, I'll fix this one. I switched to index while troubleshooting something.

Reply:
Done in 4e5901f.